docs/ota-client-guide/modules/ROOT/pages/client-provisioning-methods.adoc

```diff
@@ -81,10 +81,11 @@ image::img::device-cred-provisioning.png[width=100%]
 If you want to use "shared credential" provisioning, you don't have to do anything at all. When your account was created, we already generated a Fleet Root CA and keypair for you, and stored them on the OTA Connect server. We take the security of these keys extremely seriously: following industry best practices, they are kept in a Vault instance and only taken out when you request them.
 
 If you want to use "device credential" provisioning, you'll need to provide us with your own Fleet Root CA so that the OTA Connect server can verify devices.
```

Comment:
The indication of `own Fleet Root CA` here is intentional, because this refers to the Fleet Root CA that a user generates offline on their own and not the Fleet Root CA they obtain when they create an account. The same wording is used in other sections of the doc portal. I intend to find a better description later as it also requires a thorough edit of the developer guide.

Reply:
Hm, I see. I suggested removing "own" as "your" already expresses the idea of "own". If it triggers changes in other instructions, let's keep "own" for now.